sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Common/src/Shared/Queues/QueueListener.cs

```diff
@@ -73,7 +73,9 @@ public QueueListener(QueueClient queue,
             ConcurrencyManager concurrencyManager = null,
             string functionId = null,
             TimeSpan? maxPollingInterval = null,
-            IDrainModeManager drainModeManager = null)
+            IDrainModeManager drainModeManager = null,
+            CancellationTokenSource shutdownCancellationTokenSource = null,
+            CancellationTokenSource executionCancellationTokenSource = null)
```

Comment:
I'm assuming this new constructor has already been tested via private drop correct?